client/src/components/pages/RobotInteractionCockpit/RobotInteractionCockpit.jsx

```diff
@@ -86,13 +107,72 @@ const RobotInteractionCockpit = (match) => {
     setParameters(currentParameters);
   };
 
+  const displayStatusIcon = (status) => {
+    if (status === 'PASS' || status === 'successful') {
+      return (
+        <CheckCircleOutlined
+          style={{
+            position: 'relative',
```

Comment:
Please refactor the styles to be in a separate css file as it is more than two attributes

Reply:
That also goes for the following style tags

Reply:
The Loading Icon was not possible to refactor, because of the spin animation.